### PR TITLE
Added param to control faq / lang context refresh rate

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @lickem22 
Estimate: 1hr

## FAQ and Language Context Refresh from Db now controlled by env vars

Note that `app_secrets.env` now must have two new variables:

```
FAQ_REFRESH_FREQ=<time in secs>
LANGUAGE_CONTEXT_REFRESH_FREQ=<time in secs>
```

**TODO: We need to update deployment docs / instructions to make sure this change is captured**

We want to allow the refresh rate of FAQs and language contexts to be configurable. This changes allows environment variables to be set to configure this. 

We have chosen to have these as environment vars and not in the config files so that engineers on Praekelt's side can easily change it. Config file are "baked in" to the image and cannot be easily changed.

BUT if they'll never configure this, we SHOULD bake it in and make deployment simpler. What are your thoughts @suzinyou @lickem22?

## How Has This Been Tested?

New tests added.

## Other things to do

- Update the `app_secrets.env` file on the dev servers to ensure auto-deploy works.
    - I have done this for `AAQ` and `MC` 
    - **TODO: Need to do this for `YAL`**
- Update the `AWS secrets`
    - I have done this for `AAQ`, `MC`, and `YAL`

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the automated tests (if applicable)
- [x] I have written [good commit messages][1]
- [x] I have updated the README file (if applicable)
- [x] I have updated affected documentation (if applicable)

### CODE REVIEW

File: core_model/app/__init__.py [hunk 1]
```diff
@@ -175,9 +179,6 @@ def init_faqt_model(app):
         tags_guiding_typos=tags_guiding_typos,
     )
 
-    app.cached_faq_func = cached_faqs_wrapper(app)
```
Comment (sidravi1) [inline core_model/app/__init__.py:178]: didn't make sense for this to be in `init_faqt_model`. Moved it to `setup`

File: core_model/app/__init__.py [hunk 2]
```diff
@@ -236,18 +237,6 @@ def cached_faqs(ttl_hash):
     return cached_faqs
 
 
-def get_ttl_hash(seconds=3600):
-    """Return the same value within `seconds` time period"""
-    return time.time() // seconds
-
-
-def refresh_faqs_cached(app):
```
Comment (sidravi1) [inline core_model/app/__init__.py:244]: This is just a thin wrapper. Removed it to improve readability.

File: .github/workflows/docker-build-push.yml [hunk 3]
```diff
@@ -135,6 +135,8 @@ jobs:
           INBOUND_CHECK_TOKEN: ${{ env.CORE_INBOUND_CHECK_TOKEN }}
           ENABLE_FAQ_REFRESH_CRON: ${{ env.CORE_ENABLE_FAQ_REFRESH_CRON}}
           PROMETHEUS_MULTIPROC_DIR: ${{ env.CORE_PROMETHEUS_MULTIPROC_DIR }}
+          FAQ_REFRESH_FREQ: $${{ env.FAQ_REFRESH_FREQ }}
+          LANGUAGE_CONTEXT_REFRESH_FRESH: $${ env.LANGUAGE_CONTEXT_REFRESH_FRESH }}
```
Comment (Tanmay-97) [inline .github/workflows/docker-build-push.yml:139]: Should this be `LANGUAGE_CONTEXT_REFRESH_FREQ`? 

P.S. Just sailing around

Comment (sidravi1) [reply .github/workflows/docker-build-push.yml:139]: Yes! Thanks for picking this up

**Review by lickem22 — APPROVED** [review]
Thanks @sidravi1

**Pull request merged**